app/assets/stylesheets/custom.css.scss

```diff
@@ -1,3 +1,11 @@
+// variables
+$red: #ff0000;
+$black: #000000;
+$white: #ffffff;
+$grey: #f5f5f5;
+$blue: #5bc0de;
+$dark_blue: #337ab7;

```

Comment:
Name of variable `dark_blue` should be written in all lowercase letters with hyphens instead of underscores